logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/UserConfigRepository.kt

```diff
@@ -18,6 +18,7 @@ interface UserConfigRepository {
     fun isFileSharingEnabledFlow(): Flow<Either<StorageFailure, FileSharingStatus>>
     fun isMLSEnabled(): Either<StorageFailure, Boolean>
     fun setMLSEnabled(enabled: Boolean): Either<StorageFailure, Unit>
+    fun setClassifiedDomainsStatus(status: Boolean, domains: List<String>): Either<StorageFailure, Unit>
```

Comment:
Wouldn't maybe make more sense renaming `status` to `enabled` as it just receives a Boolean and not a ENUM of "normal" status as we do in other places?